### ISSUE TITLE
Not specifying a folder in the output notebook fails due to FileNotFound

### ISSUE THREAD

**Issue by choldgraf** [nteract/papermill#219]
I've been playing around with papermill (which is super neat) and found a bug where I needed to explicitly set the path of the output notebook to avoid an error.

Basically, if I run:

```
!papermill mynotebook.ipynb mynotebook_out.ipynb
```
then I get the error

`FileNotFoundError: output folder  doesn't exist.`

(note that there's actually a blank string after "folder")

but if I run

```
!papermill mynotebook.ipynb ./mynotebook_out.ipynb
```

Then it works fine.

Anybody else experience this? I should note that I'm on Windows subsystem for linux, so this is always a fun X-factor that breaks things sometimes :-)

**Comment by MSeal** [comment]
Ironically I pushed a PR fix for this last night: https://github.com/nteract/papermill/pull/218

**Comment by choldgraf** [comment]
so what I'm hearing is that you went backwards in time to solve this bug before I posted it?! Wow <3

**Comment by MSeal** [comment]
Haha yea. I'll get that fix merged and a new version out early next week.

**Comment by betatim** [comment]
Fixed by #218
